server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/processors/SectionNodeProcessorHelper.java

```diff
@@ -109,16 +111,14 @@ private List<VariableDefinitionNode> unwrapVariables(Node node) {
     List<Node> variables = new ArrayList<>();
     List<CopyNode> copybooks = new LinkedList<>();
 
-    node.getChildren()
-        .forEach(
-            c -> {
-              if (c.getNodeType() == NodeType.VARIABLE_DEFINITION) {
-                variables.add(c);
-              }
-              if (c.getNodeType() == NodeType.COPY) {
-                copybooks.add((CopyNode) c);
-              }
-            });
+    for (Node child : node.getChildren()) {
+      if (child.getNodeType() == NodeType.VARIABLE_DEFINITION) {
+        variables.add(child);
```

Comment:
Can we restrict the type to `VariableDefinitionNode`?

Reply:
Why?

Reply:
I am wondering if it can help simplify the processing that follows.

Reply:
I'll create a separate story for it.